processing/src/main/java/io/druid/query/metadata/metadata/SegmentMetadataQuery.java

```diff
@@ -43,17 +46,22 @@
 
     @JsonValue
     @Override
-    public String toString() {
+    public String toString()
+    {
       return this.name().toLowerCase();
     }
 
     @JsonCreator
-    public static AnalysisType fromString(String name) {
+    public static AnalysisType fromString(String name)
+    {
       return valueOf(name.toUpperCase());
     }
-  }
-
 
+    public byte[] getBytes()
```

Comment:
or probably getCacheKey in order to be consistent with other places where we create cacheKey